### PR TITLE
Update BNFabb20.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Updated the content of ms_i1.

### CODE REVIEW

File: ParisBNF/abb/BNFabb20.xml [hunk 1]
```diff
@@ -33,12 +33,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     </msIdentifier>
                     <msContents> 
                         <msItem xml:id="ms_i1">
+                            <locus from="3r" to="127v"/>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb20.xml:36]: you still need a title for ms_i1 which is still LIT3171Commen

File: ParisBNF/abb/BNFabb20.xml [hunk 2]
```diff
@@ -33,12 +33,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        <msItem xml:id="ms_i1.1">
                             <locus from="3r" to="125v"/>
                             <title type="complete" ref="LIT3171Commen"/>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb20.xml:39]: as specified in https://github.com/BetaMasaheft/Documentation/issues/2311, LIT3171Commen#Vita is a chapter inside of LIT3171Commen (which is msItem 1). 
here you can specify the relevant div (s) or insert a descriptive title, as, 
for the chapters in LIT3171Commen please check the record.

the structure is (if we follow the proposal of Dorothea @thea-m )

msItem 1 LIT3171Commen 
  msItem 1.1 commentary
  msItem 1.2 biography by Giyorgis Walda Amid


OR (if we follow what Mersha suggested in the issue)

msItem 1 Full dossier containing the commentary and the biography (then we need a new CAe ID here)
 msItem 1.1. Commentary LIT3171Commen 
 msItem 1.2 Biography by Giyorgis (then we need a new CAe ID here too, and edit other records referring to LIT3171Commen#Vita)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb20.xml [hunk 3]
```diff
@@ -34,13 +34,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     <msContents> 
                         <msItem xml:id="ms_i1">
                             <locus from="3r" to="127v"/>
+                            <title type="complete" ref="LIT3171Commen"/>
                         <msItem xml:id="ms_i1.1">
                             <locus from="3r" to="125v"/>
-                            <title type="complete" ref="LIT3171Commen"/>
+                            <title type="complete">Commentary</title>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb20.xml:40]: please extend the title to make it understandable: here Commentary on the Hebrews by John Chrysostom
below: Biography of John Chrysostom

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb20.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply ParisBNF/abb/BNFabb20.xml:40]: Done.

File: ParisBNF/abb/BNFabb20.xml [hunk 4]
```diff
@@ -33,12 +33,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                        </msItem>
-                        <msItem xml:id="ms_i2">
+                        <msItem xml:id="ms_i1.1">
+                            <locus from="3r" to="125v"/>
+                            <title type="complete">Commentary on the Hebrews by John Chrysostom</title>
+                       </msItem>
+                        <msItem xml:id="ms_i1.2">
                             <locus from="126r" to="127v"/>
-                            <title>A notice on St. John Chrysostom taken from the Tārika Walda ʾAmid, the Sənkəssār, Abušākər and the Chronicle of Yoḥannəs Madabbar.</title>
+                            <title type="complete" ref="LIT3171Commen#Vita">Biography</title>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb20.xml:44]: please write out `Biography of John Chrysostom` - or delete the word Biography, then the label from `LIT3171Commen#Vita` (`Life of John Chrysostom`) will be printed

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**